Answers.md

```diff
@@ -1 +1,18 @@
+Describe the biggest difference between .forEach & .map.
+forEach() iterates over an array and executes a provided function once each.
+While map() creates a new array with the results
 
+What is the difference between a function and a method?
+Method is a function when object is associated with it
+
+What is closure?
+Used to make global variables local
```

Comment:
Not _just_ global variables. A closure is a nested function that has access to it's outer function's variables.  Closures, store references to the outer functions variables even after function invocation has completed.